app/controllers/transfer_packages_controller.rb

```diff
@@ -4,6 +4,38 @@ class TransferPackagesController < ApplicationController
   helper_method :boxes_data
   helper_method :available_institutions
 
+  def index
+    @transfer_packages = TransferPackage
+      .within(@navigation_context.institution)
+      .includes(:receiver_institution, :sender_institution)
+      .order(created_at: :desc)
+
+    if search_uuid = params[:search_uuid].presence
+      @transfer_packages = @transfer_packages.joins(box_transfers: {box: {samples: :sample_identifiers}})
+        .where("transfer_packages.uuid LIKE concat('%', ?, '%') OR boxes.uuid LIKE concat('%', ?, '%') OR sample_identifiers.uuid LIKE concat('%', ?, '%')", search_uuid, search_uuid, search_uuid)
+        .distinct
+    end
```

Comment:
Maybe we could simplify this action with scopes on TransferPackage, taking advantage that a scope returns the original query if the lambda returns `nil`:

```ruby
scope :search_uuid, ->(uuid) {
  joins().where().distinct if uuid.present?
}

sample_transfers.search_uuid(params[:search_uuid])
```